cpython.cabal

```diff
@@ -28,8 +28,9 @@ source-repository this
   tag: haskell-cpython_3.5.0
 
 library
-  ghc-options: -Wall -O2 -fno-warn-orphans
-  cc-options: -fPIC
+  default-language:    Haskell98
+  ghc-options: -Wall -O2 -fno-warn-orphans -fno-warn-tabs
+  cc-options: -fPIC -Wno-deprecated-declarations
```

Comment:
This'll clean up the build output, sure - but really aughtn't we to fix the underlying problem rather than silence this? Maybe leaving the silencing flags out will focus us on future-proofing the codebase.